### PR TITLE
Worked on the CSS selectors content.

### PR DESCRIPTION
**Pull request by RidwanAdebosin**
I have added some contents to the slides that talks extensively on CSS selectors.

### CODE REVIEW

File: slides.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ CSS Class notes for the 1st Semester
   <button @click="$slidev.nav.openInEditor()" title="Open in Editor" class="text-xl slidev-icon-btn opacity-50 !border-none !hover:text-white">
     <carbon:edit />
   </button>
-  <a href="https://github.com/oluwasetemi/css-note" target="_blank" alt="GitHub" title="Open in GitHub"
+  <a href="https://github.com/oluwasetemi/repo" target="_blank" alt="GitHub" title="Open in GitHub"
```
Comment (Oluwasetemi) [inline slides.md]: @RidwanAdebosin let this point to the `css-note` name of the repo.

File: slides.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ CSS Class notes for the 1st Semester
...
     class="text-xl slidev-icon-btn opacity-50 !border-none !hover:text-white">
     <carbon-logo-github />
   </a>
-  <a href="https://github.com/Oluwasetemi/css-note/releases" target="_blank" alt="Download" title="Download PDF or PPTX version of the slide"
+  <a href="https://github.com/Oluwasetemi/repo/releases" target="_blank" alt="Download" title="Download PDF or PPTX version of the slide"
```
Comment (Oluwasetemi) [inline slides.md]: @RidwanAdebosin Same thing here as well.

File: slides.md [hunk 3]
```diff
@@ -52,107 +52,442 @@ CSS Class notes for the 1st Semester
...
 ### We use CSS to style our HTML elements and this is what you're going to learn througout this module.
+
 ---
 
-# Selectors and Specificity
+# Selectors
 
+Before we move deeply into Selectors, let's dive into CSS rule which is a block of code, that has one or more selectors and one or more declarations.
 
+<img src="https://web.dev/static/learn/css/selectors/image/an-image-a-css-rule-the-ced38545b4bec.svg"/>
```
Comment (Oluwasetemi) [inline slides.md]: you can add `w-<any good value&gt>` maybe with rounded-full

File: slides.md [hunk 4]
```diff
@@ -52,107 +52,442 @@ CSS Class notes for the 1st Semester
...
+
+Looking at the image in the previous slide we'll notice that CSS selector is the first part of a CSS rule. In order to choose or select HTML elements that's going to carry the CSS property values inside the rule we have to use CSS Selector. In summary, for us to add a style for a particular HTML element we need a selector.
+
+Types of selectors
+
+- Universal selector: This is also know as a wildcard, selects every single element in the document. It is represented by the asterisk character <kbd>\*</kbd>
+
+<ins>Code Example</ins>:
 
+```
```
Comment (Oluwasetemi) [inline slides.md]: Use ````css` for better formatting

File: slides.md [hunk 5]
```diff
@@ -52,107 +52,442 @@ CSS Class notes for the 1st Semester
...
 # Getting Started with CSS?
 
+### CSS which stands in for Cascading Style Sheets is a stylesheet language used to describe the presentation of a document written in HTML or XML.
 
-### CSS which stands in for Cascading Style Sheets is a stylesheet language used to describe the presentation of a document written in HTML or XML. 
 <br/>
 
 ### Just as HTML serves as the skeletal part of the web, CSS describes how the element should be rendered on the web.
 
 ### We use CSS to style our HTML elements and this is what you're going to learn througout this module.
```
Comment (Oluwasetemi) [inline slides.md]: the `througout` should be changed for `throughout`

File: slides.md
Comment (Oluwasetemi) [inline slides.md:1]: Please check the comment I left on specific lines

File: slides.md (hunk 2 above)
Comment (RidwanAdebosin) [reply slides.md]: Done

File: slides.md (hunk 1 above)
Comment (RidwanAdebosin) [reply slides.md]: I have used the repo name itself

File: slides.md (hunk 4 above)
Comment (RidwanAdebosin) [reply slides.md]: I have changed all the formatter to either ```css or ```html for better formatting.

File: slides.md (hunk 3 above)
Comment (RidwanAdebosin) [reply slides.md]: I will pass this CSS rule on the image w-200 h-100 border-10 rounded-full

**Review by Oluwasetemi — APPROVED** [review]

**Pull request merged**